Weather App/Models/Settings.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Settings.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 02.09.2021..
+//
+
+import Foundation
+
+struct Settings : Codable {
+    
+    var isCelsius : Bool
```

Comment:
Is there a reason these properties are `var`s?

Reply:
Actually, no. I will change it to let, because I will pass a new Settings model every time.